x/gamm/pool-models/balancer/amm_test.go

```diff
@@ -14,6 +15,10 @@ type BalancerTestSuite struct {
 	test_helpers.CfmmCommonTestSuite
 }
 
+func TestBalancerTestSuite(t *testing.T) {
```

Comment:
hmmm should this exist in `x/gamm/pool-models/balancer/pool_test.go`?

Reply:
I'm not sure whether this should be in

Reply:
I think it's fair it exists with wherever `BalancerTestSuite` is.
Personally would prefer both being in `pool_test.go`

Reply:
I'll move it pool_test.go